### PR TITLE
Rewrite the Tcodes description

### PR DESCRIPTION
**Pull request by gudnimg**
The old description had typos and was a bit confusing for new users

The new text is a suggestion from David Buzek at Prusa Research

### CODE REVIEW

File: Firmware/Marlin_main.cpp [hunk 1]
```diff
@@ -8628,13 +8628,13 @@ SERIAL_PROTOCOLPGM("\n\n");
...
-  @n T? Gcode to extrude shouldn't have to follow, load to extruder wheels is done automatically
-  @n Tx Same as T?, except nozzle doesn't have to be preheated. Tc must be placed after extruder nozzle is preheated to finish filament load.
-  @n Tc Load to nozzle after filament was prepared by Tc and extruder nozzle is already heated.
+  T<extruder nr.> - select extruder in case of multi extruder printer. Selects filament position 1-5 (T0-T4) in case of MMU.
+
+  For MMU2/S / MMU3:
+    T<extruder nr.> - Selects the filament position. A Gcode to load a filament to the nozzle must follow.
+    Tx - Printer asks user to select a filament position. Then loads the filament from the MMU unit into the extruder wheels only. G-code to heat up the nozzle follows.
+    Tc - Loads the filament tip from the extruder wheels into the nozzle.
+    T? - acts the same as Tx
```
Comment (leptun) [inline Firmware/Marlin_main.cpp]: Doesn't `T?` act the same way as `Tx` followed by `Tc`?

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: I think you're right. The Tcodes code isn't super obvious to me. I'll double check.

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: It looks like that's only the case when the MMU is disabled and used for single material.

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: Hold up.... the MMU2 code is calling load to nozzle. 🤔 image

I suspect we can drop this if-else then:
image

Comment (gudnimg) [reply Firmware/Marlin_main.cpp]: @leptun fixed. I'll address the potential optimisation later in a separate PR. This PR is only meant as "cleanup"

**Review by leptun — APPROVED** [review]

**Pull request merged**